### PR TITLE
Wrist -> core

### PR DESCRIPTION
**Pull request by greenblitz4590**

### CODE REVIEW

**Review by YHerm — DISMISSED** [review]

File: src/main/java/frc/robot/subsystems/wrist/Wrist.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+	public Wrist(WristStuff wristStuff) {
+		super(wristStuff.logPath());
+		this.motor = wristStuff.motor();
+		this.positionRequest = wristStuff.positionRequest();
+		this.positionSignal = wristStuff.positionSignal();
+		this.wristStuff = wristStuff;
+		this.commandsBuilder = new WristCommandsBuilder(this);
+
+		motor.resetPosition(new Rotation2d());
```
Comment (bossyshrimp68) [inline src/main/java/frc/robot/subsystems/wrist/Wrist.java:27]: Space

File: src/main/java/frc/robot/subsystems/wrist/Wrist.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+public class Wrist extends GBSubsystem {
+
+	private final ControllableMotor motor;
+	private final IRequest<Rotation2d> positionRequest;
+	private final InputSignal<Rotation2d> positionSignal;
+	private final WristStuff wristStuff;
+	private final WristCommandsBuilder commandsBuilder;
+
+	public Wrist(WristStuff wristStuff) {
+		super(wristStuff.logPath());
```
Comment (bossyshrimp68) [inline src/main/java/frc/robot/subsystems/wrist/Wrist.java:19]: Space

File: src/main/java/frc/robot/subsystems/wrist/WristState.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package frc.robot.subsystems.wrist;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public enum WristState {
+
+	IN_ARM(Rotation2d.fromDegrees(0)),
```
Comment (bossyshrimp68) [inline src/main/java/frc/robot/subsystems/wrist/WristState.java:7]: Is this the default state and the amp state? I think its better to name it default or teleop or smth

File: src/main/java/frc/robot/subsystems/wrist/factory/RealWristConstants.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+import java.util.function.Supplier;
+
+public class RealWristConstants {
+
+	private static final int POSITION_PID_SLOT = 0;
+
+	private static void configMotor(TalonSRX motor) {
+		motor.configFactoryDefault();
+
+		motor.setInverted(false);
```
Comment (bossyshrimp68) [inline src/main/java/frc/robot/subsystems/wrist/factory/RealWristConstants.java:25]: Make a constant

File: src/main/java/frc/robot/superstructure/Tolerances.java [hunk 5]
```diff
@@ -10,6 +10,7 @@ public class Tolerances {
 	public static final double TRANSLATION_VELOCITY_TOLERANCE = 0.05;
 
 	public static final Rotation2d ELBOW_POSITION_TOLERANCE = Rotation2d.fromDegrees(3);
+	public static final Rotation2d WRIST_POSITION_TOLERANCE = Rotation2d.fromDegrees(3);
```
Comment (bossyshrimp68) [inline src/main/java/frc/robot/superstructure/Tolerances.java]: Why do you need "TOLERANCE" in all the names in class terance?

**Review by bossyshrimp68 — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/subsystems/wrist/factory/RealWristConstants.java (hunk 4 above)
Comment (YHerm) [reply src/main/java/frc/robot/subsystems/wrist/factory/RealWristConstants.java:25]: no need

**Pull request merged**